### PR TITLE
real stats load from the server

### PR DESCRIPTION
**Pull request by galdd**

### CODE REVIEW

File: src/features/stats/stats.routes.ts [hunk 1]
```diff
@@ -1,65 +1,26 @@
...
-        return expense ? sum + expense.price : sum;
-      }, 0);
-      return sum + listTotal;
-    }, 0);
-
-    const uniqueUserIds = new Set(
-      expensesListData.expensesList.map((list) => list.creator.toString())
-    );
-    const totalUsers = uniqueUserIds.size;
+    const totalLists = await ExpensesListModel.countDocuments();
```
Comment (ofir-zeitoun) [inline src/features/stats/stats.routes.ts:10]: can be done in parallel?

File: src/features/stats/stats.routes.ts [hunk 2]
```diff
@@ -68,10 +29,7 @@ router.get("/", async (req, res) => {
       totalUsers,
     });
   } catch (error) {
```
Comment (ofir-zeitoun) [inline src/features/stats/stats.routes.ts:31]: remove try catch

Comment (galdd) [reply src/features/stats/stats.routes.ts:31]: done

File: src/features/stats/stats.routes.ts (hunk 1 above)
Comment (galdd) [reply src/features/stats/stats.routes.ts]: done

**Review by ofir-zeitoun — APPROVED** [review]

**Pull request merged**